patches/chromium/hack_to_allow_gclient_sync_with_host_os_mac_on_linux_in_ci.patch

```diff
@@ -0,0 +1,25 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jeremy Rose <jeremya@chromium.org>
+Date: Tue, 15 Jun 2021 15:16:12 -0700
+Subject: hack to allow gclient sync with host_os=mac on linux in CI
+
+This is a hack to work around the fact that we run `gclient sync` for
+our macOS CI on Linux.
+
+If removing this patch causes no sync failures, it's safe to delete :+1:
+
+Ref https://chromium-review.googlesource.com/c/chromium/src/+/2953903
+
+diff --git a/tools/clang/scripts/update.py b/tools/clang/scripts/update.py
+index 0f78c1ce6d8d6729052ff591bdb1b44e090baead..0b208ceeb92cb057ee427c3fa87371c8e6bf42c8 100755
+--- a/tools/clang/scripts/update.py
++++ b/tools/clang/scripts/update.py
+@@ -294,6 +294,8 @@ def main():
+       'win32': 'win',
+   }
+   default_host_os = _PLATFORM_HOST_OS_MAP.get(sys.platform, sys.platform)
++  if "host_os=mac" in os.environ.get('GCLIENT_EXTRA_ARGS', ''):
++    default_host_os = 'mac'
```

Comment:
Is there any form we could upstream this in? It seems like this script should really take `host_os` as an argument somehow?